_sources/canasta/DockerSettings.php

```
@@ -452,9 +453,20 @@ function isEnvTrue( $name ): bool {
 		$wgMainCacheType = 'redis';
 		$wgSessionCacheType = CACHE_DB;
 		break;
+	case 'CACHE_NONE':
+		$wgMainCacheType = CACHE_NONE;
```

PHPStan (2.1.51)

[error] Constant CACHE_NONE not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)